lf/demo/demo_robot.lf

```diff
@@ -2,24 +2,53 @@ target C {
     timeout: 15 sec
 };
 
-reactor Sensors {    
+// This is what we call a driver container. But I think at Luos we will have 2 drivers, one for the button and one for bumper.
+// But perhaps its more lik nodes.
+reactor Sensors {
+    // Those are like pipe beetween reactors?
     output button_out:bool;
     output bumper_out:bool;
     
+    // State are like shared variables?
+
+    // In Luos basic functionalities are represented by models (we call it template sometimes).
+    // For example "State" model is for buttons, and all the values representing the button can be output, input or both. They are associated to already maid "pipe" depending on the format of data (bool, distance, temperature, ...)
+    // To say it differently the "State" type we have already have the "output button_out:bool;" on it. And its a configurable option in this particular type.
+    // This kind of model is related to a container managing all the "pipes" of the model and sharing it with other containers.
+    // From another container, to get the value of a Button container I have to ask him a bool (we can also ask for all output data).
+    // Here is a Luos button example using models (template) : https://github.com/Luos-io/Examples/tree/template/Drivers/Button
```

Comment:
The link doesn't seem to work for me. Is it this one? https://github.com/Luos-io/Examples/tree/8986a5497804bef2f225edea8ebd4215e14e82eb/Projects/l0/Button/lib/Button

Reply:
Okay it looks like the above link is a model-free button...

Reply:
Yes that's it. This link work fine for me. This is just a piece of Luos code to demonstrate a non model based implementation of a button.